vocabularies/earth-science-data-category.ttl

```diff
@@ -98,6 +105,67 @@ esdc:geology a skos:Concept ;
     skos:notation "0403" ;
     skos:prefLabel "Geology"@en .
 
+esdc:regional-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The geological study of large-scale regions, providing a link between global and local geology, and a framework for site-specific studies. (Bally & Roberts, 2012)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Regional Geology"@en .
+
+esdc:general-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The study of many aspects of Earth Sciences, including rocks and minerals, geologic processes and hazards, and global-scale geologic systems. (Modified from: https://serc.carleton.edu/NAGTWorkshops/intro/courses/23694.html; 17.04.20020"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "General Geology"@en .
```

Comment:
General Geology should just be Geology
I would just add "General Geology" as an altLabel to the parent Geology concept.